pandas/core/frame.py

```diff
@@ -7279,32 +7279,42 @@ def f(x):
 
     def nunique(self, axis=0, dropna=True):
         """
-        Return Series with number of distinct observations over requested
-        axis.
+        Return Series with number of distinct observations.
+
+        Count distinct observations over requested axis. Can ignore NaN values.
```

Comment:
Sorry about this, but this sentence would work better as the summary i.e.

~~~
Count distinct observations over requested axis.

Return Series with number of distinct observations on the provided axis.  Can ignore NaN values.
~~~

The reason for my preference is that the summary sentence as I have swapped it is now a little less jargon-y (i.e. no mention of pandas-specific objects).  This is *slightly* nicer to read for an end-user if they just want the short version of what this method does.

Reply:
No problem :).